comments-bundle/src/Resources/contao/classes/Comments.php

```diff
@@ -296,14 +296,8 @@ protected function renderCommentForm(FrontendTemplate $objTemplate, \stdClass $o
 		// Do not index or cache the page with the confirmation message
 		if ($_SESSION['TL_COMMENT_ADDED'])
 		{
-			/** @var PageModel $objPage */
-			global $objPage;
-
-			$objPage->noSearch = 1;
-			$objPage->cache = 0;
```

Comment:
If you delete this, the page with the confirmation message will be cached. That is why this code is there.

Reply:
No it will not. There's a session cookie in your request, so the response will not be cached.

Reply:
This is probably true in this case, but most likely not in the `ModuleFaqReader` class below. Furthermore, the code also prevents that the page is indexed for searching.

Reply:
I just noticed that as well. I think we need to discuss that, we should probably adjust the search indexer to behave like the proxy (not index anything if there are cookies).

Reply:
Then I suggest you revert the changes so this PR only unsets the session variables and you create a separate issue which we can set on "up for discussion".